CONTRIBUTING.md

```diff
@@ -23,12 +23,10 @@ Thanks for your interest in contributing to Ionic's documentation! :tada: Check
 In order to run the documentation locally, install the dependencies and run the development server:
 
 ```sh
-$ npm install --legacy-peer-deps
-$ npm start
+npm install
```

Comment:
The sidebars are much smaller than they are in main

| main | branch |
| - | - |
| ![image](https://github.com/ionic-team/ionic-docs/assets/2721089/4212c764-3c41-4608-b611-9e306b073240) | ![image](https://github.com/ionic-team/ionic-docs/assets/2721089/9d1dc219-5d94-4411-8ded-b972504294c1) |

Reply:
Managed to fix it. I did notice that some words were getting cutoff behind the scroll so the width was updated. Expect the new code to show a few pixels wider.